packages/cli/templates/webcomponents/igc-ts/projects/empty/files/src/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html lang="en-GB">
+
+<head>
+  <meta charset="utf-8">
+  <style>
+    body {
+      background: #fafafa;
+    }
+  </style>
+</head>
+
+<body>
+  <div id="home"></div>
+
+  <script type="module">
+    import { html, render } from 'lit';
+    import '../dist/src/app/home/home-component.js';
+
+    const title = 'Hello owc World!';
+    render(
+      html`
+            <home-component .title=${title}>
+            some light-dom
```

Comment:
`home-component` has not `slot` and this will not be generated!

Reply:
This is actually a fallback.
I am changing it to some more meaningfull.